app/Domain/Integrations/Integration.php

```diff
@@ -53,7 +55,7 @@ public function __construct(
         public readonly string $description,
         public readonly UuidInterface $subscriptionId,
         public readonly IntegrationStatus $status,
-        public readonly IntegrationPartnerStatus $partnerStatus
+        public readonly IntegrationPartnerStatus $partnerStatus,
```

Comment:
```suggestion
        public readonly IntegrationPartnerStatus $partnerStatus
```

Reply:
I actually choose on purpose to leave these trailing comma's in.
The idea is that if we ever add a new option you would have not to change anything on the previous line, meaning that in a PR that line would not show up making the change more clear.
Happy to change it if it is not in line with the current coding standards, but it does have a purpose.

Reply:
We have enforced this rule with trailing commas on arrays but not on methods. If we want to enforce it, we could add a rule for this.

Reply:
Happy to create a rule for it later. Is it oké if I let it be for now?

Reply:
Sure